java/ql/integration-tests/java/query-suite/test.py

```diff
@@ -0,0 +1,15 @@
+import runs_on
+
+@runs_on.linux
+def test(codeql, java, cwd, expected_files, semmle_code_dir):
+    query_suites = ['java-code-quality.qls', 'java-security-and-quality.qls', 'java-security-extended.qls', 'java-code-scanning.qls']
+
+    for query_suite in query_suites:
+        actual = codeql.resolve.queries(query_suite, _capture=True).strip()
+        actual = sorted(actual.split('\n'))
```

Comment:
here is not required as we're running this on Linux, but just so you know, `actual.splitlines()` will work across platforms (accounting for Windows `\n\r` line separation)